app/javascript/components/downloads/Downloads.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import axios from 'axios';
+import { PropTypes } from 'prop-types';
+import { Button } from 'react-bootstrap';
+import reportError from '../util/ReportError';
+
+class Downloads extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  exportDownloaded = () => {
+    axios.defaults.headers.common['X-CSRF-Token'] = this.props.authenticity_token;
+    axios({
+      method: 'post',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      url: `${window.BASE_PATH}/export/downloaded`,
+      data: { id: this.props.download.id },
+    })
+      .then(() => {
+        // Maybe something like this download has been flagged for removal?
```

Comment:
is this a TODO to tell the user after download that it has been flagged for removal?